pyuvdata/utils.py

```diff
@@ -388,6 +388,59 @@ def unphase_uvw(ra, dec, uvw):
     return(xyz)
 
 
+def uvcalibrate(uvdata, uvcal, inplace=True, prop_flags=True, flag_missing=True):
+    """
+    Calibrate a UVData object with a UVCal object.
+
+    Args:
+        uvdata: UVData object
+        uvcal: UVCal object
+        inplace: bool, if True edit uvdata in place, else deepcopy
+        prop_flags : bool, if True, propagate calibration flags to data flags
+            and doesn't use flagged gains. Otherwise, uses flagged gains and
+            does not propagate calibration flags to data flags.
+        flag_missing: bool, if True, flag baselines in uvdata
+            if a participating antenna or polarization is missing in uvcal.
+
+    Returns:
+        uvdata: If not inplace, return deepcopied uvdata
```

Comment:
Please put this and all the docstrings into the numpy-style (https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)

Reply:
This this a recent change? It seems like all doc-strings in pyuvdata do not follow numpy style, or am I confused?

Reply:
We are in the process of converting, so we ask that PRs that touch existing docstrings or add new ones are in the numpy style. You'll notice that one of the PR tickboxes addresses this.